runtime/tchannel_server.go

```diff
@@ -206,7 +206,13 @@ func (s *TChannelRouter) handleHeader(
 		return ctx, err
 	}
 
-	reqUUID, ok := c.reqHeaders[s.requestUUIDHeaderKey]
```

Comment:
can you simply not use:
```
reqUUID, ok := c.reqHeaders[textproto.CanonicalMIMEHeaderKey(s.requestUUIDHeaderKey)]
```

Reply:
unfortunately, we need to standardize input map keys